src/AspNetCore/Src/inprocessstoredcontext.cxx

```diff
@@ -32,6 +34,38 @@ IN_PROCESS_STORED_CONTEXT::QueryHttpContext(
     return m_pHttpContext;
 }
 
+BOOL
+IN_PROCESS_STORED_CONTEXT::QueryIsManagedRequestComplete(
```

Comment:
the native code ideally should not touch this stored managed context. Not sure why you need it

Reply:
This doesn't touch the stored context. This is for knowing that an OnAsyncCompletion call was associated with PostCompletion(). If the context calls post completion, we should expect that the request is done and the managed request is being disposed.